app/api/api_v1/endpoints/lms_router.py

```diff
@@ -21,7 +21,7 @@ class UploadGradesBody(BaseModel):
 async def downsync(
     *,
     db: Session = Depends(get_db),
-    perm: None = Depends(PermissionDependency(UserIsInstructorPermission))
+    # perm: None = Depends(PermissionDependency(UserIsInstructorPermission))
```

Comment:
Should remove unused commented out code

Reply:
good catch, this isn't supposed to be commented.